linera-views-derive/src/util.rs

```diff
@@ -10,9 +10,18 @@ pub fn snakify(r#type: &Type) -> Ident {
     transform_type_to_ident(r#type, |s: String| s.to_case(Case::Snake))
 }
 
-/// Extracts the first `Ident` in a type and append the string 'Element' to the end.
-pub fn create_entry_name(r#type: &Type) -> Ident {
-    transform_type_to_ident(r#type, |s: String| format!("{}{}", s, "Entry"))
+/// Capitalize the first character of a string
+pub fn capitalize_first_character(s: &str) -> String {
```

Comment:
I guess this works for now but converting from snake_case to CamlCase is actually more complicated than that.
In serde-generate, I used `heck`: https://crates.io/crates/heck

Reply:
Thanks, I modify to use `heck`.